### DISCUSSION TITLE
Angular Component testing

### DISCUSSION THREAD

**Discussion by omakhalfeh** [cypress-io/cypress#15283] [Component Testing]
Hi Everyone

Thanks for this amazing work.

My question is if there is a plans or anyone knows when Cypress will be supporting the Angular components testing? 

Regards

**Comment by Makepool** [comment]
I've nothing constructive to add, only to second this request. 

I'm g̶u̶e̶s̶s̶i̶n̶g̶ hoping that once things have stabilised for Vue and React, that this can also be made available for those of us that are slumming it on Angular. Any indication that this is the case would ease my pain.

**Comment by LeJeanbono** [comment]
Hi @omakhalfeh 
You can take a look at https://github.com/bahmutov/cypress-angular-unit-test
The plan is to merge this in cypress repo.
Happy testing !

**Reply by omakhalfeh** [reply]
@LeJeanbono That’s awesome. Thank you.
Can you expect the time or know when that will happen?

**Reply by LeJeanbono** [reply]
Not actualy

**Reply by omakhalfeh** [reply]
> Not actualy

Thanks!
